cernopendata/modules/theme/assets/semantic-ui/js/records/FilesBoxApp.js

```diff
@@ -69,7 +69,7 @@ const FilesBoxApp = () => {
     <>
       {items.total > 0 && (
         <>
-          <h3>{title}</h3>
+          <h2>{title}</h2>
```

Comment:
Is this change just intended to increase the font size? Maybe we can evaluate some CSS to do that? I am not sure from a11y point of view if `h2` is recommended in this case...

Reply:
The goal was to make it consistent with the other sections of that page, which are h2:
BEFORE:
![Screenshot 2024-11-29 at 10 09 10](https://github.com/user-attachments/assets/7bde31c9-25a7-4759-9dc0-651bbf5a27f1)
AFTER:
![Screenshot 2024-11-29 at 10 09 54](https://github.com/user-attachments/assets/3de5d438-48c2-490b-bf46-b881d21b56e1)